jaxb-ri/jxc/pom.xml

```diff
@@ -113,7 +113,7 @@
                         <configuration>
                             <outputDirectory>${project.build.directory}/schemagen-cp</outputDirectory>
                             <includeArtifactIds>
-                                jakarta.activation,jakarta.xml.bind-api,
+                                jakarta.activation-api,jakarta.xml.bind-api,
```

Comment:
angus-activation should probably be added here

Reply:
Added.